content/ekscloudwatchcontainerinsights/verifycwci.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Verify CloudWatch Container Insights is working"
+chapter: false
+weight: 6
+---
+
+To verify that data is being collected in CloudWatch, launch the CloudWatch Containers UI in your browser: 
+
+https://us-east-2.console.aws.amazon.com/cloudwatch/home?region=us-east-2#cw:dashboard=Container;context=~(clusters~'eksworkshop-eksctl~dimensions~(~)~performanceType~'Service) 
+
+
+<img src="/images/ekscwci/insights.png">
```

Comment:
We shouldn't need the embedded html... can you swap to markdown format, and also move images under /static/cwci/ (or some other relevant directory name)?